temporal-sdk/src/main/java/io/temporal/internal/client/WorkflowClientRequestFactory.java

```diff
@@ -55,26 +53,30 @@ public WorkflowClientRequestFactory(WorkflowClientOptions clientOptions) {
   // If you add anything new here, keep newSignalWithStartWorkflowExecutionRequest in sync
   @Nonnull
   StartWorkflowExecutionRequest.Builder newStartWorkflowExecutionRequest(
-      WorkflowClientCallsInterceptor.WorkflowStartInput input) {
-    WorkflowOptions options = input.getOptions();
-
+      @Nonnull String workflowId,
+      @Nonnull String workflowTypeName,
+      @Nonnull io.temporal.common.interceptors.Header header,
+      @Nonnull WorkflowOptions options,
+      @Nullable Payloads inputArgs,
+      @Nullable Memo memo) {
```

Comment:
Nah. Across this whole PR I was doing work to pull usages of data converters together. "On entry points from the user code" in the call trees. Instead of spreading it all over the code. This method is cleaner now because it accepts already serialized entities and doesn't have to do it inside at all. And data conversion is done in RootWorkflowClientInvoker which is already doing data conversions anyway.